config.toml.example

```diff
@@ -582,9 +582,9 @@ changelog-seen = 2
 # Map debuginfo paths to `/rust/$sha/...`, generally only set for releases
 #remap-debuginfo = false
 
-# Link the compiler against `jemalloc`, where on Linux and OSX it should
+# Link the compiler against `mimalloc`, where on Linux and OSX it should
```

Comment:
Could we use mimalloc on Windows too? It is created by MS so it should be supported, probably.